mitmproxy/addons/proxyserver.py

```diff
@@ -211,6 +213,7 @@ def configure(self, updated) -> None:
         if "stream_large_bodies" in updated:
             try:
                 human.parse_size(ctx.options.stream_large_bodies)
+                logging.log(ALERT, "Cannot modify streamed bodies.")
```

Comment:
```suggestion
                logging.log(ALERT, "Both modify_body and stream_large_bodies are active. Streamed bodies will not be modified.")
```

(further suggestions welcome if you feel this is worse - I feel that directly linking this to the respective option names is useful here)

Reply:
I like linking to the option names.   One minor question the message now says that modify_body is active, but we are not checking for its presence.  Are we simply assuming that the body modification could happen via a def response lifecycle event or the modify_body option?

Reply:
Ah, thanks for catching this. We should definitely check if modify_body is set, and only then emit an alert! :)